config.sample.json

```diff
@@ -0,0 +1,8 @@
+{
+  "host": "example.com",
+  "db_dialect": "mariadb",
+  "db_username": "root",
+  "db_password": "secret",
+  "db_database": "mailverse",
+  "smtp_header": "example.com SMTP"
```

Comment:
why not sth. like "%s SMTP" ?
%s is then replaced with host config

Reply:
It is supposed to be fully customizable, so not require the hostname. You can do it manually.